app/views/devise/confirmations/new.html.slim

```diff
@@ -0,0 +1,11 @@
+h2
```

Comment:
if you don't need some custom devise view, you can delete it from folder /views/device... and device will choose standard one.